src/pages/About/index.tsx

```diff
@@ -237,22 +237,37 @@ export default function About() {
             <Intro>
               <IntroCopy>The leading decentralized crypto trading protocol, governed by a global community</IntroCopy>
               <ActionsContainer>
-                <InfoButton as="a" rel="noopener noreferrer" href="https://uniswap.org" target="_blank">
-                  Learn more<WrappedExternalArrow> ↗</WrappedExternalArrow>
-                </InfoButton>
-                <InfoButton as="a" rel="noopener noreferrer" href="https://docs.uniswap.org" target="_blank">
-                  Read docs<WrappedExternalArrow> ↗</WrappedExternalArrow>
-                </InfoButton>
+                <TraceEvent
+                  events={[BrowserEvent.onClick]}
+                  name={EventName.ELEMENT_CLICKED}
+                  element={ElementName.LEGACY_LANDING_PAGE_LINK}
+                >
+                  <InfoButton as="a" rel="noopener noreferrer" href="https://uniswap.org" target="_blank">
+                    Learn more<WrappedExternalArrow> ↗</WrappedExternalArrow>
+                  </InfoButton>
+                </TraceEvent>
+                <TraceEvent
+                  events={[BrowserEvent.onClick]}
+                  name={EventName.ELEMENT_CLICKED}
+                  element={ElementName.DOCS_LINK}
+                >
+                  <InfoButton as="a" rel="noopener noreferrer" href="https://docs.uniswap.org" target="_blank">
+                    Read docs<WrappedExternalArrow> ↗</WrappedExternalArrow>
+                  </InfoButton>
+                </TraceEvent>
               </ActionsContainer>
             </Intro>
           </Panels>
           <CardGrid>
-            {CARDS.map(({ darkBackgroundImgSrc, lightBackgroundImgSrc, ...card }) => (
-              <Card
+            {CARDS.map(({ darkBackgroundImgSrc, lightBackgroundImgSrc, elementName, ...card }) => (
+              <TraceEvent
```

Comment:
These events are not logging because the browser event is happening inside the <Card /> component, pls move TraceEvent to inside <Card />

Reply:
will do